README.md

```diff
@@ -490,6 +490,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
   * [Squidex](https://squidex.io/) - Offers free tier for small projects. API / GraphQL first. Open source and based on event sourcing (versing every change automatically).
   * [InstaWP](https://instawp.com/) - Launch a WordPress site in a few seconds. A free tier with 5 Active Sites, 500 MB Space, 48 hrs Site Expiry.
   * [Storyblok](https://www.storyblok.com) - A Headless CMS for developers and marketers that works with all modern frameworks. The Community (free) tier offers Management API, Visual Editor, ten sources, Custom Field Types, Internationalization (unlimited languages/locales), Asset Manager (up to 2500 assets), Image Optimizing Service, Search Query, Webhook + 250GB Traffic/month included.
+  * [PayloadCMS](https://payloadcms.com/) - a free and open-source headless CMS aimed sharply at providing the best, developer-first experience possible to build out APIs and admin UI.
```

Comment:
For this to qualify there need to be a cloud free tier, please add details of that in the description

Reply:
@ripienaar ,I sign up for the plaform and didn't find any **free plan** instead they offer **7 day trial.**

Their FAQ section also says that their will be a free plan **But not yet.** .It more like a **self hosted** product **for now**

![Screenshot from 2023-12-20 19-36-45](https://github.com/ripienaar/free-for-dev/assets/97863234/34fe0cde-7419-47ad-a1da-f55398c26b37)


![Screenshot from 2023-12-20 19-29-49](https://github.com/ripienaar/free-for-dev/assets/97863234/bf70ce30-65d8-4226-a6de-da82c8b57bce)

Reply:
Thank you, will close for now.